examples/mnist/quantized.cpp

```diff
@@ -30,10 +33,11 @@
 using namespace tiny_dnn;
 using namespace tiny_dnn::activation;
 
-void construct_net(network<sequential>& nn) {
-    // connection table [Y.Lecun, 1998 Table.1]
+void construct_net(network<sequential> &nn) {
+// connection table [Y.Lecun, 1998 Table.1]
 #define O true
 #define X false
+    // clang-format off
```

Comment:
It's very non trivial to create such a rule. But `clang-format` is open-source, so you're welcome to try. Generally I believe the resulting code style isn't worse than what we had and it's uniform.